Renode Error

When running

scripts/setup
cd proj/example_cfu
make renode

I get following error: 
`
21:34:35.8609 [INFO] Loaded monitor commands from: /home/halteproblem/Dokumente/CFU-Playground/third_party/renode/scripts/monitor.py

21:34:36.4144 [INFO] Including script: /home/halteproblem/Dokumente/CFU-Playground/proj/example_cfu/build/renode/digilent_arty.resc

21:34:36.4279 [INFO] System bus created.

21:34:37.0805 [WARNING] SimulationFilePath not set!

21:34:37.6377 [INFO] digilent_arty: GDB server with all CPUs started on port :10001

21:34:37.7012 [ERROR] cpu.cfu0: Error while opening dynamic library: /home/halteproblem/Dokumente/CFU-Playground/proj/example_cfu/build/renode/libVtop.so: undefined symbol: tlib_try_interrupt_translation_block
`

--- a/scripts/setup
+++ b/scripts/setup
@@ -115,13 +115,17 @@ elif ! ${RENODE_DIR}/renode --version | grep "${RENODE_VERSION_NO}.*${RENODE_VER
     echo "Warning: Your Renode version (in ${RENODE_DIR}) does not match the required one (${RENODE_VERSION})"
 fi
 
+# After this commit, a dependency on 'tlib' was added that we don't support here.
+# This caused issue #805.  Pinning to this version fixes it.
+RENODE_INFRA_VERSION_SHA=509a073
+
 # Get necessary files from Renode VerilatorPlugin to build CFU library for Renode tests
 # It is a temporary workaround as long as the sources are not present in Renode portable
 VIL_DIR=${RENODE_DIR}/verilator-integration-library
 if [ ! -e "${VIL_DIR}" ]; then
     mkdir -p ${VIL_DIR}/src/buses
-    wget -O ${VIL_DIR}/renode_imports.h https://raw.githubusercontent.com/renode/renode-infrastructure/master/src/Emulator/Cores/renode/include/renode_imports.h
-    wget -O ${VIL_DIR}/renode_imports_generated.h https://raw.githubusercontent.com/renode/renode-infrastructure/master/src/Emulator/Cores/renode/include/renode_imports_generated.h
+    wget -O ${VIL_DIR}/renode_imports.h https://raw.githubusercontent.com/renode/renode-infrastructure/${RENODE_INFRA_VERSION_SHA}/src/Emulator/Cores/renode/include/renode_imports.h
+    wget -O ${VIL_DIR}/renode_imports_generated.h https://raw.githubusercontent.com/renode/renode-infrastructure/${RENODE_INFRA_VERSION_SHA}/src/Emulator/Cores/renode/include/renode_imports_generated.h
     wget -O ${VIL_DIR}/src/renode_cfu.cpp https://raw.githubusercontent.com/renode/renode/${RENODE_VERSION_SHA}/src/Plugins/VerilatorPlugin/VerilatorIntegrationLibrary/src/renode_cfu.cpp
     wget -O ${VIL_DIR}/src/renode_cfu.h https://raw.githubusercontent.com/renode/renode/${RENODE_VERSION_SHA}/src/Plugins/VerilatorPlugin/VerilatorIntegrationLibrary/src/renode_cfu.h
     wget -O ${VIL_DIR}/src/renode.h https://raw.githubusercontent.com/renode/renode/${RENODE_VERSION_SHA}/src/Plugins/VerilatorPlugin/VerilatorIntegrationLibrary/src/renode.h

